docs/components/contributors.vue

```diff
@@ -228,15 +261,19 @@ export default {
     processBackers(backers = []) {
       // Backers are provided in reverse chronological order
       // so we sort by larger amount first, then by date
-      // Limit to top 16 backers
-      this.backers = backers.sort(this.sortCompare).slice(0, MAX_BACKERS)
+      // Limit to top N backers
+      // this.backers = backers.sort(this.sortCompare).slice(0, MAX_BACKERS)
```

Comment:
@tmorehouse Can we get rid of this commented out code?

Reply:
Yeah, I just had it in there in case the following code didn't work ;-)

Reply:
fixed